app/code/core/Mage/Adminhtml/controllers/System/AccountController.php

```diff
@@ -67,6 +65,10 @@ public function saveAction()
             $user->setPasswordConfirmation($this->getRequest()->getParam('password_confirmation', false));
         }
 
+        $backend_locale = $this->getRequest()->getParam('backend_locale', false);
+        $backend_locale = $backend_locale == 0 ? null : $backend_locale;
+        $user->setBackendLocale($backend_locale);
```

Comment:
What is this horrible variable name? `$backend_locale` => `$backendLocale` no?

Reply:
ehmm in other PRs I had to switch to camelcase (which I never liked) and now the otherway around?

example:
<img width="195" alt="Screenshot 2023-03-16 alle 20 47 00" src="https://user-images.githubusercontent.com/909743/225748929-def41c57-df80-45e3-b3f2-f625de1f0dca.png">

Reply:
or:
<img width="484" alt="Screenshot 2023-03-16 alle 20 47 43" src="https://user-images.githubusercontent.com/909743/225749097-7522fdf2-368c-4dc6-9a47-bde4d0d2f8f5.png">

Reply:
I'm doing too many things at the same time, apologies @luigifab 

fixing it

Reply:
... I vote for `$backendLocale` like `$orderItem`.

Reply:
it has to be camelcase since it's the standard, although it's not my cup of tea.

it was already used in this PR, I just had the wrong one in one of the files